CONTRIBUTING.md

```diff
@@ -120,7 +128,7 @@ from the main (upstream) repository:
 - Check out the beta branch:
```

Comment:
this is inconsistent now - also idk if we need a section for after the PR, but we might as well make it a script they could just execute

Reply:
Yeah, the section might be unnecessary, we copied this I think from Angular contributing guidelines. Could do with a bit more polishing.